src/components/util/Rivet.tsx

```diff
@@ -149,9 +149,9 @@ const parseRivetHidden = (hide?: Hidden) => ({
  */
 export type ComponentDecorator<T extends Props> = ( props: T ) => string;
 
-export const rivetize = <T extends React.HTMLAttributes<HTMLElement>>(Component: React.ComponentType<T>): React.SFC<T & Props> =>
-({ className, border, display, hide, margin, padding, typescale, ...attrs }: Props & React.HTMLAttributes<HTMLElement>) => (
-        <Component className={classnames(
+export const rivetize = <T extends React.HTMLAttributes<HTMLElement>>(Component: React.ComponentType<any>): React.FC<T & Props> =>
+    ({ className, border, display, hide, margin, padding, typescale, ...attrs }: T & Props) => (
```

Comment:
Is the default spacing for `.tsx` files 2 spaces or 4?

Reply:
🤷🏻‍♂️. We should develop a strategy for automatic formatting and linting.